Calculator/Calculator/Operator.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  Operator.swift
+//  Calculator
+//
+//  Created by Hyungmin Lee on 2023/06/02.
+//
+
+enum Operator: Character, CaseIterable, CalculateItem {
+    case add = "+"
+    case subtract = "−"
+    case divide = "/"
+    case multiply = "*"
+    
+    func calculate(lhs: Double, rhs: Double) -> Double {
+        let result: Double
+        
+        switch self {
+        case .add:
+            result = add(lhs, rhs)
+        case .subtract:
+            result = subtract(lhs, rhs)
+        case .divide:
+            result = divide(lhs, rhs)
+        case .multiply:
+            result = multiply(lhs, rhs)
+        }
+        
+        return result
+    }
+    
+    private func add(_ lhs: Double,_ rhs: Double) -> Double {
```

Comment:
밑에 네개의 메서드도 동일하게 띄어쓰기 오타가 있습니다..! 😅
```suggestion
    private func add(_ lhs: Double, _ rhs: Double) -> Double {
```